### PR TITLE
Parallel coordinate plot

### PR DESCRIPTION
**Pull request by Muthenya**

### CODE REVIEW

File: instat/dlgParallelCoordinatePlot.Designer.vb [hunk 1]
```diff
@@ -23,17 +23,169 @@ Partial Class dlgParallelCoordinatePlot
     <System.Diagnostics.DebuggerStepThrough()> _
     Private Sub InitializeComponent()
         Dim resources As System.ComponentModel.ComponentResourceManager = New System.ComponentModel.ComponentResourceManager(GetType(dlgParallelCoordinatePlot))
+        Me.ucrSelectorParallelCoordinatePlot = New instat.ucrSelectorByDataFrameAddRemove()
+        Me.cmdParcoordOptions = New System.Windows.Forms.Button()
+        Me.cmdPlotOptions = New System.Windows.Forms.Button()
+        Me.UcrReceiverMultiple1 = New instat.ucrReceiverMultiple()
```
Comment (maxwellfundi) [inline instat/dlgParallelCoordinatePlot.Designer.vb]: This does not seem to have the proper naming convention

File: instat/dlgParallelCoordinatePlot.Designer.vb [hunk 2]
```diff
@@ -23,17 +23,169 @@ Partial Class dlgParallelCoordinatePlot
...
         Dim resources As System.ComponentModel.ComponentResourceManager = New System.ComponentModel.ComponentResourceManager(GetType(dlgParallelCoordinatePlot))
+        Me.ucrSelectorParallelCoordinatePlot = New instat.ucrSelectorByDataFrameAddRemove()
+        Me.cmdParcoordOptions = New System.Windows.Forms.Button()
+        Me.cmdPlotOptions = New System.Windows.Forms.Button()
+        Me.UcrReceiverMultiple1 = New instat.ucrReceiverMultiple()
+        Me.lblXVariables = New System.Windows.Forms.Label()
+        Me.ucrReceiverFactor = New instat.ucrReceiverSingle()
+        Me.lblFactor = New System.Windows.Forms.Label()
+        Me.ucrBase = New instat.ucrButtons()
+        Me.ucrSaveGraph = New instat.ucrSaveGraph()
```
Comment (maxwellfundi) [inline instat/dlgParallelCoordinatePlot.Designer.vb]: You need to change this to ucrsave. We stopped using ucrsavegraph

Comment (Muthenya) [reply instat/dlgParallelCoordinatePlot.Designer.vb]: Okay. Let me get to it immediately.

Comment (dannyparsons) [reply instat/dlgParallelCoordinatePlot.Designer.vb]: @maxwellfundi let's delete this ucrSaveGraph control since we don't want to use it anymore.

**Comment by maxwellfundi** [comment]
What are we using instead?

**Comment by maxwellfundi** [comment]
I think I had an earlier comment for that to be changed maybe it was not
seen but I can change that to ucrsave

On Dec 6, 2017 14:46, fundimaxwell@gmail.com wrote:

What are we using instead?

On Dec 6, 2017 14:16, "Danny Parsons" <notifications@github.com> wrote:

*@dannyparsons* commented on this pull request.
------------------------------

In instat/dlgParallelCoordinatePlot.Designer.vb
<https://github.com/africanmathsinitiative/R-Instat/pull/4263#discussion_r155208389>
:

> @@ -23,17 +23,169 @@ Partial Class dlgParallelCoordinatePlot
     <System.Diagnostics.DebuggerStepThrough()> _
     Private Sub InitializeComponent()
         Dim resources As
System.ComponentModel.ComponentResourceManager = New
System.ComponentModel.ComponentResourceManager(GetType(dlgParallelCoordinatePlot))
+        Me.ucrSelectorParallelCoordinatePlot = New
instat.ucrSelectorByDataFrameAddRemove()
+        Me.cmdParcoordOptions = New System.Windows.Forms.Button()
+        Me.cmdPlotOptions = New System.Windows.Forms.Button()
+        Me.UcrReceiverMultiple1 = New instat.ucrReceiverMultiple()
+        Me.lblXVariables = New System.Windows.Forms.Label()
+        Me.ucrReceiverFactor = New instat.ucrReceiverSingle()
+        Me.lblFactor = New System.Windows.Forms.Label()
+        Me.ucrBase = New instat.ucrButtons()
+        Me.ucrSaveGraph = New instat.ucrSaveGraph()

@maxwellfundi <https://github.com/maxwellfundi> let's delete this
ucrSaveGraph control since we don't want to use it anymore.

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/africanmathsinitiative/R-Instat/pull/4263#discussion_r155208389>,
or mute the thread

**Comment by dannyparsons** [comment]
I mean we can remove ucrSaveGraph from the project completely since we're not using it and it can just confuse developers.

**Comment by maxwellfundi** [comment]
Yeah, I understand. Sending a pull request for that separately

**Pull request merged**